internal/repositories/storage.go

```diff
@@ -0,0 +1,71 @@
+package repositories
+
+import (
+	"errors"
+	"os"
+	"strings"
+)
+
+var ListURL = make(map[string]string)
+
+type Storage interface {
+	AddShort(string, string) error
+	GetFullURL(string) (string, error)
+}
+
+type FileStorage struct {
+	FileStorage string
+}
+
+type InMemoryStorage struct {
+}
+
+func (f FileStorage) GetFullURL(id string) (string, error) {
+	var u string
+
+	data, err := os.ReadFile(f.FileStorage)
+	if err != nil {
+		//http.Error(w, err.Error(), http.StatusInternalServerError)
+		return u, err
+	}
+	foundURL := false
+	for _, line := range strings.Split(string(data), "\n") {
+		if strings.HasPrefix(line, id) {
+			u = strings.Join(strings.Split(line, ":")[1:], ":")
+			//log.Print(u)
+			foundURL = true
+			break
```

Comment:
можно обойтись без буленовых переменных:
```suggestion
			return strings.Join(strings.Split(line, ":")[1:], ":"), nil

```